pom.xml

```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.varchenko</groupId>
-    <artifactId>InsertSort</artifactId>
+    <artifactId>BubbleSort</artifactId>
```

Comment:
This is not a `BubbleSort` project anymore. 
Pick some valid name. 

E.G. Sortings and algorithms.

Reply:
done